pkg/health/etcdmember/check_ready.go

```diff
@@ -73,7 +73,7 @@ func (r *readyCheck) Check(ctx context.Context, etcd druidv1alpha1.Etcd) []Resul
 		renew := lease.Spec.RenewTime
 		if renew == nil {
 			r.logger.Info("Member hasn't acquired lease yet, still in bootstrapping phase", "name", lease.Name)
-			continue
+			return []Result{}
```

Comment:
Why is this needed?
Won't returning an empty result set if any lease has an empty `RenewTime` result in an empty member list?

Reply:
Couldn't get you what you asked